6_nodejs/week1/avg.js

```diff
@@ -0,0 +1,24 @@
+// Function to calculate average of inputs with process
+function averageInput(){
+    const inputs = process.argv.slice(2)
+    let total = 0;
+    if (inputs.length < 3) {
+        console.log('Please type three numbers.');
+    } else {
+        let allNumbers = true;
```

Comment:
it is generally safer/more readable to initiate your boolean variables as `false` rather then as `true`